core/src/epicli/data/common/ansible/playbooks/roles/kafka/tasks/setup-kafka.yml

```diff
@@ -119,9 +119,6 @@
     owner: "{{ specification.kafka_var.user }}"
     group: "{{ specification.kafka_var.group }}"
     mode: 0644
-    src: log4j.properties
-  notify:
-    - restart kafka
```

Comment:
why removing notify?

Reply:
This task was previously a "template" ansible task, somebody converted it to a "file" ansible task, but chnaged only the "tamplate" string to a "file" string. It's completely incorrect, and this notify has no meaning, it is just a bug tbh :(
If you provide (for "file" module) "dest" and "src" dont set "state: link" it is big warning in 2.8, but fatal error in 2.10.